nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core-api/src/main/java/org/apache/nifi/controller/AbstractConfiguredComponent.java

```diff
@@ -57,6 +57,7 @@ public AbstractConfiguredComponent(final ConfigurableComponent component, final
         this.component = component;
         this.validationContextFactory = validationContextFactory;
         this.serviceProvider = serviceProvider;
+        this.name = new AtomicReference<>(component.getClass().getSimpleName());
```

Comment:
@olegz - This "kind of" changes the apparently undocumented behavior of the name field. It used to be initialized to null, now it is initialized with a value. Rather than have ```component.getClass().getSimpleName()``` here, would it make more sense to have ```setName(processor.getClass().getSimpleName()); ``` in the constructor of StandardProcessorNode?